### PR TITLE
Test/unittests

### PR DESCRIPTION
**Pull request by jlunenburg**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Start by adding python-mock as test-dependency. If then the version is incorrect, we can change the target from debian(1.3.0) to pip(2.0.0).

**Comment by LoyVanBeek** [comment]
The version I have locally is 2.0.0, so then we'd need the pip version. Can you make a target for that in tue-env?

**Comment by LoyVanBeek** [comment]
The tests weren't OK yet, but at least now our code is tested instead of failing on something 'unrelated'. Thnx!

File: src/conversation_engine/engine.py [hunk 1]
```diff
@@ -246,27 +250,30 @@ class ConversationEngine(object):
...
+        :type command_target: str
         :param give_examples: Include examples when talking with the user.
             These are randomly generated from the active part of the grammar
         :type give_examples: bool
         """
 
         self._state = ConversationState()
 
-        self._action_client = Client(robot_name)
+        self._action_client = action_client  # ToDo: update (GPSR) dependency before merging!
```
Comment (LoyVanBeek) [inline src/conversation_engine/engine.py]: https://github.com/tue-robotics/tue_robocup/pull/726

**Pull request merged**